app/functions/sortingCompaniesByAmountOfUsers.js

```diff
@@ -0,0 +1,8 @@
+
+export default function sortingCompaniesByAmountOfUsers (list) {
+    list.sort((a,b) => {
+        if (a.usersOfSpecificCompany.length < b.usersOfSpecificCompany.length) { return -1; }
+        if (a.usersOfSpecificCompany.length > b.usersOfSpecificCompany.length) { return 1; }
+        return 0;
+    });
```

Comment:
We have discussed sometime befor about sorting algorithm, it can be simplified a lot, take a look at [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description)